tests/time_tests/scripts/run_timetest.py

```diff
@@ -61,8 +61,8 @@ def prepare_executable_cmd(args: dict):
         str(args["executable"].resolve(strict=True)),
         "-m", str(args["model"].resolve(strict=True)),
         "-d", args["device"],
-        "-ip", args["input_precision"],
-        "-op", args["output_precision"],
+        "-ip", args.get("input_precision", ""),
+        "-op", args.get("output_precision", ""),
```

Comment:
```suggestion
        "-op" if args["output_precision"] else ""
```

Reply:
With previous implementations "-op" will always be provided to executable, but the key is optional